root/static/scripts/common/MB/Control/Autocomplete.js

```diff
@@ -762,6 +762,11 @@ MB.Control.autocomplete_formatters = {
                      _.escape(commaOnlyList(comment)) + ')</span>');
         }
 
+        if (item.typeName)
+        {
+            a.append('<br /><span class="autocomplete-comment"> Type: ' + _.escape(item.typeName) + '</span>');
```

Comment:
About your suggestion to "do the usual with types and move languages to a 'Languages: X, Y' line like writers / artists," I guess the question is whether languages are more important than types for works?

Anyway, if we keep the display you have here, the "Type:" part needs to be fixed so that it's translated.